source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -74,6 +74,18 @@ class ExtAuthzLoggingInfo : public Envoy::StreamInfo::FilterState::Object {
     upstream_host_ = std::move(upstream_host);
   }
 
+  bool hasFieldSupport() const { return true; }
+  Envoy::StreamInfo::FilterState::Object::FieldType getField(absl::string_view field_name) const {
+    if (field_name == "latency" && latency_) {
```

Comment:
for clarity
```suggestion
    if (field_name == "latency" && latency_.has_value()) {
```

Reply:
What are the units of latency? Should it be renamed `latency_ms` or `latency_us` so it's clear?